pkg/exporter/azureblob_exporter.go

```diff
@@ -33,23 +29,18 @@ func NewAzureBlobExporter(creationTime, hostname string) *AzureBlobExporter {
 }
 
 func createContainerURL() (azblob.ContainerURL, error) {
-	APIServerFQDN, err := utils.GetAPIServerFQDN()
-	if err != nil {
-		return azblob.ContainerURL{}, err
-	}
+	accountName := os.Getenv("AZURE_BLOB_ACCOUNT_NAME")
+	sasKey := os.Getenv("AZURE_BLOB_SAS_KEY")
+	containerName := os.Getenv("AZURE_BLOB_CONTAINER_NAME")
 
-	containerName := strings.Replace(APIServerFQDN, ".", "-", -1)
-	containerLen := strings.Index(containerName, "-hcp-")
```

Comment:
💡 This logic needs to be extracted in some kind of git issue, (cc: @sophsoph321  - because `az-cli` will now do this before supplying the container name as `APIServerName` ) also fyi: @rzhang628 and @qpetraroia  - `vscode` also need similar tracking workitem in its queue and I can work on that change.

Reply:
yes, this needs to be documented in the release 0.5. Not sure what would be the benefit of an issue

Reply:
Cool, so the reason we need this is because periscope as a tool from beginning  has working contract with “az-cli” via kollect command and latter vscode started using so anything effecting long term to these tools or any change of contract needs to be highlighted.

with tracking workitem we need to inform those consuming tools: (give enough information for this change) 

refrence:

* https://github.com/Azure/aks-periscope#dependent-consuming-tools-and-working-contract
* https://github.com/Azure/aks-periscope#user-guide

thanks 🙏

Reply:
I think these 2 paragraphs have to be removed from our README:
- first one: we are not developping aks-periscope only for vscode & az cli; some others might use it and anyway noone can change this file unless a PR is approved. + we already documented this file as deprecated
- 2nd one: the detail of how to use an extension of az-cli shall be in the extension repo. We are not going to watch the extension repo so that we can align our README. However we can indicate that az-cli can be used to deploy aks-periscope and add a link to our repo to the extension